sirepo/package_data/static/html/raydata-analysis-queue.html

```diff
@@ -1,5 +1,6 @@
 <div class="container-fluid">
   <div class="col-md-12">
     <div data-basic-editor-panel="" data-view-name="analysisQueue"></div>
+    <div data-basic-editor-panel="" data-view-name="historyScans"></div>
```

Comment:
Any reason why this is named historyScans? I think a name like recentlyExecutedScans would match up better with what the model represents and would be easier to track through the code (model name is similar to report title). Likewise it would look more similar to our other models representing similar data (executedScans and queuedScans)

Reply:
71d7b187ccf540a8b22a16f86cc3ece57952d92d